build.sh

```diff
@@ -416,6 +417,58 @@ function build_java()
 
 }
 
+################################################################################
+function clean_documentation()
+{
+    # Remove the content of ${doc_folder}/_build
+    rm -rf ${doc_folder}/_build
+}
+
+function build_documentation()
+{
+    clean_documentation
+
+    # Generate HTML
+    echo ""
+    echo -e "${INFO_TAG} Generating HTML documentation"
+    cd ${doc_folder}
+    ${MAKE_EXE} -f Makefile html > /dev/null 2>&1
+    if [ "$?" != 0 ]; then
+        echo -e "${ERROR_TAG} Failure generating HTML documentation"
+        echo -e "${ERROR_TAG} You will need to install:
+            sudo pip install -U sphinx
+            sudo pip install sphinx_rtd_theme"
+        clean_documentation
+        exit -1
+    fi
+    rm -rf ${doc_folder}/doc/html
+    mkdir -p ${doc_folder}/doc/html
+    cp -rf ${doc_folder}/_build/html ${doc_folder}/doc/
+    echo -e "${INFO_TAG} HTML Generation successful. You will find it under:
+        ${doc_folder}/doc/html/index.html"
+
+
+    # Generate PDF
+    echo ""
+    echo -e "${INFO_TAG} Generating PDF documentation"
+    cd ${doc_folder}
+    ${MAKE_EXE} -f Makefile latexpdf > /dev/null 2>&1
+    if [ "$?" != 0 ]; then
+        echo -e "${ERROR_TAG} Failure generating PDF documentation"
+        echo -e "${ERROR_TAG} You will need to install:
```

Comment:
"On Linux systems you might need to install \"texlive-full\"."

Reply:
Done